File: app/src/main/java/com/remotearthsolutions/expensetracker/contracts/AccountDialogContract.java
package com.remotearthsolutions.expensetracker.contracts;

import com.remotearthsolutions.expensetracker.databaseutils.models.AccountModel;

import java.util.List;

public interface AccountDialogContract {
    interface View {
        void onAccountFetchSuccess(List<AccountModel> accounts);

        void onAccountFetchFailure();
    }
}

File: app/src/main/java/com/remotearthsolutions/expensetracker/contracts/AccountDialogContract.kt
package com.remotearthsolutions.expensetracker.contracts

import com.remotearthsolutions.expensetracker.databaseutils.models.AccountModel

interface AccountDialogContract {
    interface View {
        fun onAccountFetchSuccess(accounts: List<AccountModel>?)
        fun onAccountFetchFailure()
    }
}